problems/variables.js

```diff
@@ -2,47 +2,85 @@
 
 // Age Calculator:
 //  * Store your birth year in a constant variable.
+const myBirthday = 1984
 //  * Store a future year in a variable.
+let futureYear = 2021
 //  * Calculate your 2 possible ages for that year based on the stored values.
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
+  let ageMin = (futureYear - myBirthday)
+  let ageMax = ((futureYear - myBirthday) + 1)
```

Comment:
You mixed these up. Max age is future - birthday and min is future - birthday - 1

Reply:
let ageMax = (futureYear - myBirthday)
let ageMin = ((futureYear - myBirthday) - 1)